geotrek/api/v2/serializers.py

```diff
@@ -788,6 +767,28 @@ def get_departure_city(self, obj):
             city = zoning_models.City.objects.all().filter(geom__contains=geom).first()
             return city.code if city else None
 
+        def _replace_image_paths_with_urls(self, data):
+            def replace(html_content):
+                if not html_content:
+                    return html_content
+                soup = BeautifulSoup(html_content, features="lxml")
+                imgs = soup.find_all('img')
+                for img in imgs:
+                    if img.attrs['src'][0] == '/':
+                        base_url = self.context.get("request").build_absolute_uri("/")[:-1]
+                        img['src'] = base_url + img.attrs["src"]
+                # Note: beautifulsoup makes a valid HTML doc from the data,
+                # but we don't want <html> or <body> in the output
+                return "".join([str(c) for c in soup.find("body").children])
```

Comment:
I think there is an option or method in beautifulsoup for this

Reply:
Right my complicated statement is useless. By passing `features="html.parser"` to BeautifulSoup constructor no extra HTML tags are added to the input (no <html> or <body>).